modules/ROOT/pages/mergetags.adoc

```diff
@@ -0,0 +1,68 @@
+= Merge Tags plugin
+:navtitle: Merge Tags
+:description: Enables Merge Tags functionality in the {productname} editor
+:keywords: merge, tags, tag, mergetags, mergetag, mergetags_list, mergetags_prefix, mergetags_suffix, mail merge, personalization token
+:pluginname: Merge Tags
+:plugincode: mergetags
+:pluginminimumplan: tiertwo
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.2v.adoc[]
+
+The {pluginname} plugin allows the user to insert a merge tag (also known as a personalization token, or a mail merge field).
+
+Merge tags can be inserted as-you-type; selected from a drop-down list when a specified prefix is typed; or selected and inserted from the searchable Merge tags toolbar menu button.
+
+Once a merge tag is inserted, the plugin leaves a non-editable variable wrapped with a prefix and suffix, making it easily identifiable.
+
+== Interactive example
+
+liveDemo::merge-tag[]
+
+include::partial$misc/purchase-premium-plugins.adoc[]
+
+== Basic setup
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'mergetags',
+  toolbar: 'mergetags',
+});
+----
+
+The {pluginname} plugin provides an xref:autocompleter.adoc[autocompleter] for adding a merge tag without using the toolbar button or menu item.
+
+The autocompleter is triggered by typing a specified prefix, controlled by the xref:#mergetags_prefix[`+mergetags_prefix+`] option.
+
+By default, this prefix is `{{`.
+
+Entering characters after the prefix begins filtering the merge tags list.
+
+== Usage notes
+
+. Merge tag are non-editable but can be inline-formatted.
++
+For example, a merge tag can be set to any available typeface and type-size and can be set to *bold*, or _italic_.
+
+. Merge tags can be changed.
++
+Select the Merge tag, click the Merge tag toolbar button, and select a new tag from the menu that presents.
+
+. Text that matches an existing Merge tag will be recognised as a Merge tag when it is pasted or otherwise inserted into a {productname} document.
++
+For example, if `Sender.Firstname` is a Merge tag value, and the Merge tag prefix and suffix have their default values, adding the string, `{{Sender.FirstName}}`, to a {productname} document will result in the string automatically being recognised as a Merge tag.
+
```

Comment:
```suggestion
== Using Merge Tags

. Merge tag contents are non-editable but can have any inline-formats applied to them.
+
For example, a merge tag can be set to any available typeface, type-size, foreground or background color, or can be set to *bold*, or _italic_.

. Merge tags can be changed.
+
A selected merge tag can be changed to any other merge tag by using the merge tags toolbar menu button.

. Text that matches an existing merge tag will be recognised as a merge tag when it is pasted or otherwise inserted into a {productname} document.
+
Content containing the specified prefix and suffix, and matching a specified merge tag, will be inserted as a merge tag when pasted into the editor. For example, if `Sender.Firstname` is a Merge tag value, and the Merge tag prefix and suffix have their default values, adding the string, `{{Sender.FirstName}}`, to a {productname} document will result in the string automatically being recognised as a Merge tag.

. Merge tags can be nested within the `+mergetags_list+` option.
+
The `+mergetags_list+` option allows for the specification of a nested menu item within the merge tags toolbar menu button. This option allows any number of nested menus and items for merge tag categorisation.

```
Some edits and added details about nested menus. We should be consistent with capitalisation of "merge tags", I believe lowercase is suitable when talking about merge tags as they exist in the editor, however should be capitalised when referencing the plugin name (and using {pluginname}).